core/tauri-utils/src/lib.rs

```diff
@@ -211,6 +211,108 @@ impl Default for Env {
   }
 }
 
+/// Size unit type. Effective to size related settings
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+pub enum SizeUnit {
```

Comment:
This should be renamed `PixelUnit` and used for both size and poistion. No need to have two different enums.